tgui/packages/tgui/interfaces/ShuttlePicker.tsx

```diff
@@ -0,0 +1,102 @@
+import { resolveAsset } from '../assets';
+import { useBackend } from '../backend';
+import { Box, Button, Flex, Section } from '../components';
+import { Window } from '../layouts';
+
+type Shuttle = {
+  name: string;
+  description: string;
+  ref: string;
+};
+
+type ShuttlePickerData = {
+  name: string;
+  desc: string;
+  assetpath: string;
+  current_ref: string;
+  shuttles: Shuttle[];
+};
+
+export const ShuttlePicker = (_props) => {
+  const { act, data } = useBackend<ShuttlePickerData>();
+
+  return (
+    <Window width={500} height={500} title={'Shuttle Selection'}>
+      <Window.Content>
+        <Flex height={'100%'} direction={'row'}>
+          <Flex.Item width={'45%'}>
+            <ShuttleSelection />
+          </Flex.Item>
+          <Flex.Item width={'55%'}>
+            <Box // Wrap the entire right column in a Box to keep the layout consistent
+              style={{
+                display: 'flex',
+                flexDirection: 'column',
+                height: '100%',
```

Comment:
Why did you change here? none of this is screenshotted or documented in your pr

Reply:
Mostly just added comments and renamed some things to be more generic but that specifically was to make the contents of the window properly fill and center. Was triggering my OCD.